### ISSUE TITLE
Lab: Working with Relational Data Stores in the Cloud - select sample

### ISSUE THREAD

**Issue by Datachick** [MicrosoftLearning/DP-200-Implementing-an-Azure-Data-Solution#145]
Exercise 2, task 1

> On the Additional settings tab, select **sample** for Use existing data. Click Review + Create then Create.

In that line, _sample_ should be bolded since it's an value that needs to be set.  Students are missing this because it's in plaintext.

**Comment by Datachick** [comment]
This would also be a great place to give a head's up that deploying a Dedicated SQL pool can take several minutes (10-15?).

**Comment by Kaijisse** [comment]
Lab 5 Exercise 1, Task 1, number 4, is where you click on additional settings and sample. I have bolded sample ( see screenshot). 
![image](https://user-images.githubusercontent.com/38757647/119987555-735efe80-bfc5-11eb-966c-d4c6907830d7.png)

In relation to deploying a dedicated SQL pool and how many minutes it takes (for me 7 min), I've added a comment under Exercise 2, Task 1, nr 10. See screenshot:
![image](https://user-images.githubusercontent.com/38757647/119987800-b6b96d00-bfc5-11eb-8c4b-aca4a75f31f2.png)

Closing the issues.
